docs/src/DeveloperDocumentation/printing_details.md

```diff
@@ -16,30 +16,40 @@ e.g. to print the `base_ring(X)` when in `one line` mode.
 It exists to make sure that `one line` stays compact and human readable.
 
 Top-level REPL printing of an object will use `detailed` mode by default
-```julia
+```julia-repl
 julia> X
 detailed
 ```
 Inside nested structures, e.g. inside a `Vector`, the `one line` mode is used.
-```julia
+```julia-repl
 julia> [X,X]
 3-element Vector{TypeofX{T}}
-one line
-one line
-one line
+ one line
+ one line
+ one line
 ```
 
 #### An Example for the 2 + 1 print modes
+
+detailed mode:
+```jldoctest
+julia> E = elliptic_curve(QQ, [-82, 0])
+Elliptic curve
+  over rational field
+with equation
+  y^2 = x^3 - 82*x
 ```
-# detailed
-General linear group of degree 24
-  over Finite field of degree 7 over GF(29)
 
-# one line
-General linear group of degree 24 over GF(29^7)
+one line mode:
+```jldoctest
```

Comment:
these need to be named doctests so that this can access the `E` from the previous one